src/firefly/js/core/background/BackgroundMonitor.css

```diff
@@ -7,8 +7,8 @@
     display: flex;
     flex-direction: column;
     justify-content: space-between;
-    width: 330px;
-    min-width: 330px;
+    width: 500px;
+    min-width: 500px;
```

Comment:
Changing this setting will affect the original layout.  It was designed to support single, double, and triple columns display.  Notice the max-width of 590px below?  Changing min to 500 would limit the range from 500-590 which does not make much sense anymore.  Before I look into how this setting affect the outcome old the original use-case, can I ask why it was necessary to change it in the first place?